<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Marco Accorinti</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>1</horstretch>
    <verstretch>1</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Consolas</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>trackmania-manager</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/resources/icons/app.ico</normaloff>:/resources/icons/app.ico</iconset>
  </property>
  <widget class="QWidget" name="central_widget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>360</width>
     <height>240</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTableWidget" name="matchsettings_table">
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <column>
       <property name="text">
        <string>Challenge</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Status</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="_2">
      <item>
       <spacer name="spacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="add_tracks_button">
        <property name="text">
         <string>Add track(s)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="remove_tracks_button">
        <property name="text">
         <string>Remove track(s)</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="file_menu">
    <property name="font">
     <font>
      <family>Consolas</family>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="open_file_action"/>
    <addaction name="save_file_action"/>
    <addaction name="close_file_action"/>
    <addaction name="separator"/>
    <addaction name="edit_settings_action"/>
    <addaction name="separator"/>
    <addaction name="exit_action"/>
   </widget>
   <widget class="QMenu" name="help_menu">
    <property name="font">
     <font>
      <family>Consolas</family>
      <stylestrategy>PreferDefault</stylestrategy>
     </font>
    </property>
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="about_action"/>
   </widget>
   <addaction name="file_menu"/>
   <addaction name="help_menu"/>
  </widget>
  <widget class="QStatusBar" name="status_bar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
    </font>
   </property>
  </widget>
  <action name="open_file_action">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/icons/open-file.ico</normaloff>:/resources/icons/open-file.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;Open file</string>
   </property>
   <property name="toolTip">
    <string>Open a matchsettings file</string>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="save_file_action">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/icons/save-file.ico</normaloff>:/resources/icons/save-file.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;Save file</string>
   </property>
   <property name="toolTip">
    <string>Save current matchsettings file</string>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="close_file_action">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/icons/close-file.ico</normaloff>:/resources/icons/close-file.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;Close file</string>
   </property>
   <property name="toolTip">
    <string>Close current matchsettings file</string>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="exit_action">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit application</string>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="about_action">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/icons/about.ico</normaloff>:/resources/icons/about.ico</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>Information about this application</string>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
    </font>
   </property>
  </action>
  <action name="edit_settings_action">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/icons/edit-settings.ico</normaloff>:/resources/icons/edit-settings.ico</iconset>
   </property>
   <property name="text">
    <string>E&amp;dit settings</string>
   </property>
   <property name="toolTip">
    <string>Edit application settings</string>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
    </font>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
